src/items.zig

```diff
@@ -454,11 +454,21 @@ const ToolPhysics = struct { // MARK: ToolPhysics
 		for(tool.type.properties()) |property| {
 			var sum: f32 = 0;
 			var weight: f32 = 0;
+			for(0..25) |i| {
+				weight += property.weights[i];
```

Comment:
This breaks the averaging, which only averages active properties. With this it averages all properties.

I'd prefer if you did the normalization during loading to avoid confusion in this function.

Reply:
What do you mean by "active properties"?

Should I only normalize the weights during load for parameters with method `.sum`?